Location.js

```diff
@@ -1,9 +1,8 @@
 class Location {
   // prettier-ignore
-  constructor(locationid, displayName, responderPhoneNumber, movementThreshold, durationTimer, stillnessTimer, heartbeatSentAlerts, heartbeatAlertRecipients, doorCoreId, radarCoreId, radarType, reminderTimer, fallbackTimer, twilioNumber, fallbackNumbers, initialTimer, alertApiKey, isActive, firmwareStateMachine, sirenParticleId, client) {
+  constructor(locationid, displayName, movementThreshold, durationTimer, stillnessTimer, heartbeatSentAlerts, heartbeatAlertRecipients, doorCoreId, radarCoreId, radarType, reminderTimer, fallbackTimer, twilioNumber, fallbackNumbers, initialTimer, isActive, firmwareStateMachine, sirenParticleId, client) {
```

Comment:
once we move fallbackNumbers and heartbeatAlertRecipients, we'll end up with a clients/devices table setup that conforms to the schema we designed a few months ago, right?

No particular reason to do it here and now though.

Reply:
Yes :smile: We're getting closer step-by-step (https://docs.google.com/document/d/1k-xkWnR7xoWRi6gEiN8p-nda5bhxDvOqNYCtPNi7OwY/edit). I thought about doing the fallbackNumbers during this PR, but it touched too many un-related places and I didn't want to make it too confusing.